server/index.js

```diff
@@ -44,21 +65,34 @@ app.get('/', 'home', (req, res) => {
   res.render('index');
 });
 
-app.post('/add-interviewer', 'add-interviewer', async (req, res) => {
-  debugServer('POST /add-interviewer', req.body);
-
-  await sequelize.models.user.createInterviewer(req.body);
-
-  res.redirect(app.namedRoutes.build('home'));
-});
-
-app.post('/add-applicant', 'add-applicant', async (req, res) => {
-  debugServer('POST /add-interviewer', req.body);
-
-  await sequelize.models.user.createApplicant(req.body);
+async function saveToDb(data, reqType) {
+  if (reqType === 'add-applicant') {
```

Comment:
We store incoming form data in session and only after OAUTH occurs we check if data is really exists and start saving to db. It occurs in oauth callback on different route anyway. So, it would be better if we define only one route for purpose of creating users and dispatch on user role. Maybe even add some sort of REST in our app.

But if we want to work with current app setting I see no way how we can stay with two routes without dispatch.